industrial/kinetic/install_pkg_script.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+packagelist=(
+    libcap2-bin
+    libsvm-dev
+    libudev-dev
+    libv4l-dev
+    libpcl-dev
+    libproj-dev
+    python-catkin-tools
+    python-pygraphviz
+    python-rosinstall
+    python-setuptools
+    python-zmq
+    python-gtk2
+    python-gobject
+    python-wxgtk2.8
+    g++-multilib
+    flex
+    mongodb
+    librealsense2
+    librealsense2-dkms
+    librealsense2-gl
+    librealsense2-utils
+    ros-kinetic-amcl
+    ros-kinetic-bfl
```

Comment:
If I might make a suggestion, it might be better to list the dependencies in the corresponding `package.xml` and use `rosdep` to install this.

Reply:
Yes you are right and it is already there in repository.debs in the mas_industrial_repo, however I just put here so that there is no need to  install these packages each time a container is started. I wanted it to be pre-installed in the container.

Reply:
The dependencies would only be installed once when the image is built (e.g. by Travis or by the person building locally). Afterwards, any container using the image would have the dependencies installed.  

Maybe I'm misunderstanding the purpose of the docker image, you want to manually pre-install these dependencies but not build the industrial repos?

Reply:
Yes, I just want an image with all the dependencies in it so that I can later volume mount the code when I start the container. The purpose of the docker image is to have a proper environment for software development, as most of the times we have missing dependencies and python issues when we build on our laptops locally.

Reply:
I think that’s a stronger case for using rosdep! Clone the industrial repo, run rosdep to install dependencies that match your code and not a separate script. Otherwise you’ll need to keep the dependencies updated on both places!

Reply:
ok. I will do that. But is it fine to have an initial version like this ?

Reply:
In fact, if you migrate to `rosdep` instead of using `repository.debs` you won’t need two different scripts to install dependencies in `melodic` and `kinetic` (which will also get outdated depending on which is the main branch).

Reply:
I wouldn’t merge this to `master`

Reply:
Yes thats fine. I will for now create a branch and ensure travis is working, so that an initial version of these images are pushed to docker hub. Then i will do the changes you mentioned so that it can be merged to master. Does this work for you?

Reply:
Sounds good

Reply:
I have removed the bash script which manually installs the ros packages and added rosdep to install the dependencies. Both the branch in this repo and the branch in my fork have been updated. Please let me know if any other changes are required before it can be merged to master.